cleaners/adobe_reader.xml

```diff
@@ -27,8 +30,28 @@
     <label>Cache</label>
     <description>Delete the cache</description>
     <action command="delete" search="walk.files" path="~/.adobe/Acrobat/*/Cache"/>
-    <action command="delete" search="walk.files" path="$localappdata\Adobe\Acrobat\*\Cache"/>
-    <action command="delete" search="glob" path="$USERPROFILE\AppData\LocalLow\Adobe\Acrobat\9.0\Search\*.idx"/>
+    <action command="delete" search="walk.files" path="$LocalAppData\Adobe\Acrobat\*\Cache"/>
+    <action command="delete" search="glob" path="$UserProfile\AppData\LocalLow\Adobe\Acrobat\9.0\Search\*.idx"/>
+    <action command="delete" search="glob" path="$UserProfile\AppData\Local\Adobe\Acrobat\10.0\Cache\*"/>
```

Comment:
This line and others look redundant to `$localappdata\Adobe\Acrobat\*\Cache` that was already there

Reply:
The version on my PC looks a bit different then the version you have mentioned... Is it the latest file?

But I see I can remove some lines if I work with paths like `$localappdata\Adobe\Acrobat\*\Cache`... I will do this and make a new push.